frontend/src/components/DownloadAnalysisLink/BuildCSV.ts

```diff
@@ -99,6 +99,7 @@ export function getStaticCsvUrl(selectedDataset: Dataset | null, method: Method)
     return ''
   }
   const datasetString = kebabCase(selectedDataset.name)
+  const state = selectedDataset.state
   const methodString = kebabCase(method.toString())
-  return `${staticCsvRootUrl}${staticCsvPath}${datasetString}-${methodString}.csv`
+  return `${staticCsvRootUrl}${staticCsvPath}${datasetString}-${state}-${methodString}.csv`
```

Comment:
@ericboucher we could also use `getStaticDatasetString` from `api.ts`, unless there was a reason you avoided doing this?

Reply:
No I think it is a good idea to use it. I just tried to NOT make too many changes at once, although I know it didnt seem like this is what I was doing haha